content/blog/standup.md

```diff
@@ -124,6 +124,23 @@ Any tool that publishes data is good enough.
 
 Many teams don't even use any! (These are very mature teams. I would recommend having some visual representation.)
 
+##### Q: Our team is small, do I even need a standup meeting?
+
+If you have some mechanism like a storyboard or sticky notes on the wall,
+and you know the project status without having a separate meeting,
+you don't even need a standup meeting in theory.
+
+However, I recommend having a standup meeting even if you are a two-person team.
+
+Sometimes, it helps to wear a specific hat to think about abstract project status; a standup meeting helps there.
+
+There is a famous unrelated terminology called "duck debugging."
+When stuck in a problem,
+a developer tries to explain the situation to a rubber duck, and a solution automatically emerges.
+In the same way, if we do a standup meeting as if we are publishing project status,
+even though the receiver either already knows or doesn't even want to know,
+you will get solutions to your problems.
```

Comment:
I'm not sure about duck debugging here. I agree that there should be standup even in a two person team to promote collaboration and track progress but duck debugging makes you go inside the technical problem often which is not the idea of a standup?

Or do you mean that since this is a small enough team, they can talk about any problems in detail as well?

Reply:
Duck debugging is to give example of wearing a different hat, so instead of thinking about the problem and finding in the code, you explain to someone - that way you wear a different hat. The same way, even if you are developing and pairing all the time, you are never in explanation hat! CHanging the hat give tremendous results. 

Do you want to articulate it in a different way?

Reply:
I believe we should mention this "wearing a different hat idea" explicitly. It's not implicitly conveying it if I had not read your comment, Akshay.
Otherwise, the new write up looks good!